src/EventHandler/ConfigurableEventHandlerEntityBundle.php

```diff
@@ -30,14 +64,34 @@ public static function determineQualifiedEvents(Event $event, $event_name, array
    * {@inheritdoc}
    */
   public function summary() {
-    // Nothing to do by default.
+    $bundle = $this->configuration['bundle'];
+    $bundle_label = isset($this->bundlesInfo[$bundle]['label']) ? $this->bundlesInfo[$bundle]['label'] : $bundle;
+    $suffix = isset($bundle) ? ' ' . t('of @bundle-key %name', array('@bundle-key' => $this->entityInfo->getBundleLabel(), '%name' => $bundle_label)) : '';
```

Comment:
never split up translatable messages like this. Use 2 distinct t() calls instead.

Oh, and don't use t(), us $this->t() instead.